e2e/pages/adf/searchFiltersPage.ts

```diff
@@ -153,4 +154,20 @@ export class SearchFiltersPage {
         Util.waitUntilElementIsVisible(this.showLessButtonForSize);
     }
 
+    clickPngImageType() {
+        Util.waitUntilElementIsVisible(this.pngImageFileType);
+        return this.pngImageFileType.click();
+    }
+
+    getBucketNumberOfFilterType(fileType, filterLength) {
+        let fileTypeFilter = element(by.css('mat-checkbox[data-automation-id="checkbox-SEARCH.FACET_FIELDS.' + fileType + '"] span'));
+        Util.waitUntilElementIsVisible(fileTypeFilter);
+        let bucketNumber = fileTypeFilter.getText().then((valueOfBucket) => {
+            let numberOfBucket = valueOfBucket.substring(filterLength, valueOfBucket.length - 1);
```

Comment:
Cn you use instead of filterLength the last character of filter.name (in this case '(') ?

Reply:
That is not working either, it requires the length of the total characters that are not needed (in this case "PNG Image (" ), not the starting point of the characters that are needed.